lib/slide_hub/convert_util.rb

```diff
@@ -30,7 +30,7 @@ def ppm_to_jpg(dir)
     end
 
     def jpg_to_thumbnail(list)
```

Comment:
__[reek]__
* [DataClump] takes parameters [dir, file] to 4 methods [:ref](https://github.com/troessner/reek/blob/master/docs/Data-Clump.md)